### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by diegombeltran**
Initial release. Colonies and Venux Next are missing, but we're working on adding them soon. Some strings could be missing, it's still work in progress. We'd like to have feedback from the Spanish players so we can find them sooner.

Spanish flag image is also missing (Language selector at start menu)

### CODE REVIEW

File: src/locales/cn/cards.json [hunk 1]
```diff
@@ -120,7 +120,6 @@
 
     "Ants": "蚂蚁",
     "Action: Remove 1 Microbe from any card to add 1 to this card.": "行动: 从任意的卡上移除一个微生物资源,并放置到本卡上",
-    "1 VP PER 2 MICROBES ON THIS CARD.": "这张卡上每有2个微生物资源,获得1分",
     "Requires 4% oxygen. 1 VP per 2 Microbes on this card.": "需要至少4%的氧气浓度。这张卡上每有2个微生物资源,获得1分",
 
     "Release of Inert Gases": "释放惰性气体",
```
Comment (alrusdi) [inline src/locales/cn/cards.json:125]: Spanish translation shouldn't contain any other translations. Please use empty string as placeholder

**Comment by vsrisuknimit** [comment]
> Spanish flag image is also missing (Language selector at start menu)

Check out: assets/flags_responsive.png. Not sure if this is the correct flag. But you can add.

```
.language-icon--es {
    background-position: 2px 26.86%;
}
```

**Comment by diegombeltran** [comment]
Hi, thank you for your feedback. I just bought Venux Next and Colonies, so we will translate all those strings and do as @alrusdi said.

We will also look into the css to set the flag up.

**Comment by diegombeltran** [comment]
Hello. I added Venus/Colonies, new promo cards and refactored strings a little.
I have two questions:
- I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
- The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.

**Comment by alrusdi** [comment]
>     * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> 

Yes, empty string is ignored in all translations 

>     * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.

Do you have a demo server to look at it?

**Comment by diegombeltran** [comment]
> > ```
> > * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> > ```
> 
> Yes, empty string is ignored in all translations
> 
> > ```
> > * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.
> > ```
> 
> Do you have a demo server to look at it?

I sent you a PM

**Comment by diegombeltran** [comment]
> > ```
> > * I tried to leave "" as a placeholder for not-yet-finished strings (Mostly Ares, Moon and some promo cards) Not sure if I understood right but does leaving and empty string as a placeholder fallback to English?
> > ```
> 
> Yes, empty string is ignored in all translations
> 
> > ```
> > * The Spanish flag looks a bit odd with that emblem. I personally don't care but it may be more accurate removing it to avoid confusion.
> > ```
> 
> Do you have a demo server to look at it?

I do, but I rather not share it publicly. Can I send you an email?

**Comment by bafolts** [comment]
@alrusdi I don't want to merge this if you are waiting for something. It seems good to merge to me. Are you waiting for any change?

**Review by bafolts — APPROVED** [review]

**Comment by alrusdi** [comment]
> @alrusdi I don't want to merge this if you are waiting for something. It seems good to merge to me. Are you waiting for any change?

No, I don't wait anything. Let's merge it!

**Comment by diegombeltran** [comment]
Hi, @alrusdi , what is your Discord nickname? Thanks!

**Comment by alrusdi** [comment]
> Hi, @alrusdi , what is your Discord nickname? Thanks!

the same as here - **alrusdi**

**Pull request merged**